cloud/deploy/deploy.go

```diff
@@ -191,96 +257,50 @@ func Deploy(deployInput InputDeploy, client astro.Client) error { //nolint
 	deploymentURL := "cloud." + domain + "/" + deployInfo.workspaceID + "/deployments/" + deployInfo.deploymentID + "/analytics"
 
 	if deployInput.Dags {
-		if deployInput.Pytest != "" {
-			version, err := buildImage(&c, deployInput.Path, deployInfo.currentVersion, deployInfo.deployImage, deployInput.ImageName, deployInfo.dagDeployEnabled, client)
+		if !deployInfo.dagDeployEnabled {
+			return fmt.Errorf(enableDagDeployMsg, deployInfo.deploymentID) //nolint
```

Comment:
try this pattern for errors:

```
       // define this var up top
        var errDagDeployNotEnabled = errors.New(enableDagDeployMessage)
	// use it on line 261 like:
	return fmt.Errorf("%w: %s", errDagDeployNotEnabled, deployInfo.deploymentID)
```
This way you do not need the `//nolint`

Reply:
This doesnt work, to what we want to display. The above snippet, prints the message as

```
Error: DAG-only deploys are not enabled for this Deployment. Run 'astro deployment update %s --dag-deploy enable' to enable DAG-only deploys.
: cl84ycf6j15232cw3v0p0q2eb
```

I am going to not worry about this, since we need to fix the linting anyway once we upgrade the linter